example/deployments/cumulus/jl-tf.tfvars

```diff
@@ -41,5 +41,5 @@ key_name      = "jl"
 include_orca = true
 drop_database = "False"
 
-distribution_url = "https://jwhwz6qg4j.execute-api.us-east-1.amazonaws.com:7000/DEV/"
+tea_distribution_url = "https://e5wdgj4q4i.execute-api.us-east-1.amazonaws.com/DEV/"
```

Comment:
```suggestion
tea_distribution_url = "https://jwhwz6qg4j.execute-api.us-east-1.amazonaws.com:7000/DEV/"
```

Reply:
Ah yeah, I think I replaced this with mine, sorry about that. I'm curious: why do you have these variables in `deployments/cumulus/jl-tf.tfvars`? Is it so that your CI stack is more easily testable if you want to do something like manually test auth? (assuming this is your CI stack, could be wrong)

Reply:
I use the same stack for local build and testing.

Reply:
Cool, that makes sense. I might update my tfvars so I can do the same in a future PR 😄